servicemesh-install/topics/tutorial-kiali.adoc

```diff
@@ -38,27 +40,36 @@ image:kiali-login.png[Login Page]
 +
 Log in to the Kiali console using the user name and password that you specified in the custom resource file during installation.
 
+[[kiali-overview-page]]
+== Overview page
+
+After you log in you see the Overview page, which provides you with a quick overview of the health the various namespaces in your system.
+
+image:kiali-overview.png[Overview Page]
+
 [[kiali-graph-page]]
 == Graph page
 
-After you log in you see the Graph page.  This page shows a graph with all the microservices, connected by the requests going through them. On this page, you can see how the services interact with each other.
+. Click the *Graph* link in the left navigation.  The Graph page shows a graph with all the microservices, connected by the requests going through them. On this page, you can see how the services interact with each other.
 
-. From the *Namespace* menu, select Bookinfo.  Now the graph displays only the services in the Bookinfo application.
+. From the *Namespace* menu, select `bookinfo`.  Now the graph displays only the services in the Bookinfo application.
+. Click the *Legend* button in the lower left.  Kiali displays a pop-up with the graph legend.
```

Comment:
s/Click the *Legend* button in the lower left./Click \*Legend\* in the lower left \<noun\>. 

("For other interface elements... use only the label,
title, descriptive name, or name that is indicated by the tooltip. However, if an inter-
face contains elements with the same label or very similar labels, include the element
type to ensure clarity. If you refer to a button as a button, you must distinguish the
type of button, such as a push button or radio button.")

s/pop-up with/window that contains